docs/fundamentals/networking/http/httpclient-guidelines.md

```diff
@@ -88,6 +91,9 @@ The preceding code:
 - Passes the `socketHandler` to the `resilienceHandler` with the retry logic.
 - Instantiates an `HttpClient` given the `resilienceHandler`.
 
+> [!IMPORTANT]
+> The `Microsoft.Extensions.Http.Resilience` library is currently marked as experimental and it may change in the future.
```

Comment:
Do you want to link to [this page](https://learn.microsoft.com/dotnet/csharp/language-reference/attributes/general#experimental-attribute)?